DesignMotivations.md

```diff
@@ -1,22 +1,31 @@
-## Roles
+## Roles and epics
 
 #### QM software developer
-I write code that performs QM calculations and produces output for its end users
+Role: I write code that performs QM calculations and produces output for its end users
+ - I want users to be able to visualize orbitals using only basis set info and MO coefficients, so that I don't have to build the infrastructure to dump gaussian cube files
```

Comment:
Could this be more like "I want users to take advantage of the full capabilities of modern visualization software, including...".